docker/entrypoint.sh

```diff
@@ -2,20 +2,20 @@
 
 set -e
 user=ipfs
-repo="$IPFS_PATH"
 
 if [ `id -u` -eq 0 ]; then
+    echo "Changing user to $user"
     # ensure directories are writable
-    su-exec "$user" test -w "$repo" || chown -R -- "$user" "$repo"
-    su-exec "$user" test -w "$IPFS_CLUSTER_PATH" || chown -R -- "$user" "$IPFS_CLUSTER_PATH"
+    su-exec "$user" test -w "${IPFS_PATH}" || chown -R -- "$user" "${IPFS_PATH}"
```

Comment:
Again a reminder from countzen2000's comments, we are not changing the assumption of existence of su-exec.

Reply:
But su-exec exists in alpine and go-ipfs image so... I am also assuming /bin/sh exists to run this script... it's not a script for a system where su-exec doesn't exist...

Reply:
@countzen2000 FYI

Reply:
Actually, su-exec is built by hand in the go-ipfs image, but we build on top so it's fine as long as they don't change it. I haven't looked into the implications of not using it though (same as tini)

Reply:
Hi. 

I'm using the image ipfs/ipfs-cluster:latest and I just got this error:
/usr/local/bin/entrypoint.sh: line 13: su-exec: not found

su-exec is only used when this condition is met:
"if [ `id -u` -eq 0 ]; then"

Can someone check if this actually works correctly ?